MetaMask/metamask-extension
# Update TextField Imports

## **Description**

Update TextField imports to no longer use the deprecated version. 
Update the types if needed such as `TextFieldType` and `TextFieldSize`

## **Related issues**

Fixes: #21870 

## **Manual testing steps**

1. Run locally 
2. View TextField component
3. Input text/number to confirm no errors

## **Screenshots/Recordings**

_**Videos attached to component files**_

### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
